php/example_code/class_examples/tests/TestClassExamples.php

```diff
@@ -0,0 +1,20 @@
+<?php
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace ClassExamples;
+
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @group integ
+ */
+class TestClassExamples extends TestCase
+{
+    public function testCommandPoolRuns()
+    {
+        include_once __DIR__ . "/../CommandPool.php";
```

Comment:
I assume that including this file means it runs the code in it so this is actually doing something?

Reply:
yes, PHP can "do stuff" and "define stuff." Including this here with do both types of actions.